docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+api:
+    image: docker.digital.homeoffice.gov.uk/lev-api:v1.2.0-14
```

Comment:
Do we want this in the repo?
I thought this was to be stuff that's all out in the open? This would require special credentials.